morf-integration-test/src/test/resources/log4j.properties

```diff
@@ -9,3 +9,5 @@ log4j.appender.CONSOLE.layout.ConversionPattern=%d{yyyy-MM-dd HH:mm:ss} %-5p %c{
 log4j.logger.org.alfasoftware.morf.diagnostics.DatabaseAndSystemTimeConsistencyChecker=DEBUG
 
 log4j.logger.org.alfasoftware.morf.jdbc.AbstractConnectionResources=INFO
+
+log4j.logger.org.alfasoftware.morf.jdbc.DatabaseMetaDataProvider=DEBUG
```

Comment:
Was this an intentional change?

Reply:
It was an attempt to increase code coverage. It failed. And I meant to revert it.